src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
 public class Application {
+
```

Comment:
remove this redundant empty line please
we don't need empty lines after class declarations or method signatures